src/web/services/student.service.ts

```diff
@@ -129,10 +132,34 @@ export class StudentService {
    * Loads list of students from a course in CSV format by calling API.
    */
   loadStudentListAsCsv(queryParams: { courseId: string }): Observable<string> {
-    const paramsMap: Record<string, string> = {
+    return this.httpRequestService.get(ResourceEndpoints.COURSE, {
       courseid: queryParams.courseId,
-    };
-    const responseType: string = 'text';
-    return this.httpRequestService.get(ResourceEndpoints.STUDENTS_CSV, paramsMap, responseType);
+      entitytype: 'instructor',
+    }).pipe(mergeMap((course: Course) => {
+      return this.httpRequestService.get(ResourceEndpoints.STUDENTS, {
+        courseid: queryParams.courseId,
+      }).pipe(map((students: Students) => {
+        return this.processStudentsToCsv(course.courseId, course.courseName, students.students);
+      }));
+    }));
+  }
+
+  processStudentsToCsv(courseId: string, courseName: string, students: Student[]): string {
+    const csvRows: string[][] = [];
+    csvRows.push(['Course ID:', StringHelper.removeExtraSpace(courseId)]);
+    csvRows.push(['Course Name:', StringHelper.removeExtraSpace(courseName)]);
+    csvRows.push([]);
+    csvRows.push(['Section', 'Team', 'Full Name', 'Last Name', 'Status', 'Email']);
```

Comment:
I'll close one eye of the section filtering (i.e. we will not display section column if all students are from the default section)